tests/runtime/test_http_server.py

```diff
@@ -240,6 +251,235 @@ def test_services_disabled(open_port, enabled_services):
             # )
 
 
+## Functional Tests #######################################################################
+
+
+def test_build_dm_object_simple(runtime_http_server):
```

Comment:
love all the new tests! (and definitely want them separated form the http server ones later 😄 )